src/plugins/saved_search/common/saved_searches_utils.ts

```diff
@@ -6,36 +6,45 @@
  * Side Public License, v 1.
  */
 
+import { SerializedSearchSourceFields } from '@kbn/data-plugin/common';
 import { SavedSearch, SavedSearchAttributes } from '.';
+import { SerializableSavedSearch } from './types';
 
-export const fromSavedSearchAttributes = (
+export const fromSavedSearchAttributes = <
```

Comment:
Ultimately, I default to your opinion here :) I did this so that I **didn't** have to reserialize, since that felt worse to me... It would have to be reserialized in the `deserialize` method for the embeddable, and I was trying to keep that method as fast as possible. But if that is what you would prefer here, I'm good with that too!